src/DaPigGuy/PiggyCustomEnchants/enchants/weapons/LifestealEnchant.php

```diff
@@ -20,14 +20,6 @@ class LifestealEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Lifesteal";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 18, so I commented this line